ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+# must of these are expected by other steps, but we write them here to make sure
+# we have all of the required environmental variables before proceeding
+#echo ${CONFIG_TYPE} > ${SHARED_DIR}/CONFIG_TYPE
+#echo ${CONFIG_API_VERSION} > ${SHARED_DIR}/CONFIG_API_VERSION
+#echo ${CLUSTER_TYPE} > ${SHARED_DIR}/CLUSTER_TYPE
+#echo ${OPENSTACK_EXTERNAL_NETWORK} > ${SHARED_DIR}/OPENSTACK_EXTERNAL_NETWORK
+#echo ${OPENSTACK_COMPUTE_FLAVOR} > ${SHARED_DIR}/OPENSTACK_COMPUTE_FLAVOR
+#echo ${OS_CLOUD} > ${SHARED_DIR}/OS_CLOUD
+#echo "${BASE_DOMAIN}" > ${SHARED_DIR}/BASE_DOMAIN
+#echo ${KURYR_ENABLED} > ${SHARED_DIR}/KURYR_ENABLED
+
+# We have to truncate cluster name to 14 chars, because there is a limitation in the install-config
+# Now it looks like "ci-op-rl6z646h-65230".
+# We will remove "ci-op-" prefix from there to keep just last 14 characters. and it cannot start with a "-"
+UNSAFE_CLUSTER_NAME=${NAMESPACE}-${JOB_NAME_HASH}
```

Comment:
Where do we use `UNSAFE_CLUSTER_NAME` and `SAFE_CLUSTER_NAME`?

Reply:
CLUSTER_NAME is used in several places: when creating the install-config.yaml, when creating fips, when creating dns rescords. etc. etc.  But the variable is not available from the ENV. It needs to be derived from the ENV variables NAMESPACE and JOB_NAME_HASH,  But as the note says that might be a problem if it is more than 14 characters long (according to the notes, maybe is not the case anymore). 
Either way, instead of figuring out in each step what it could be, we use the ipi-conf-openstck-command.sh to set it in stone for the test. Then every step can reference that. It would be better to throw it up into the ENV but env actually do not carry accross steps, so we have to write it to file system to make it available.
we can probably do without writing UNSAFE_CLUSTER_NAME. Its useful for debugging the test, but once it works there is probably no need to keep it around.

Reply:
Oh, I missed that we were writing the variable `$SAFE_CLUSTER_NAME` to `${SHARED_DIR}/CLUSTER_NAME`. All good, thanks. I think it's ok to leave `${SHARED_DIR}/UNSAFE_CLUSTER_NAME` there for debugging too if you want to add it back.